packages/api/src/platforms/vtex/resolvers/aggregateOffer.ts

```diff
@@ -2,32 +2,54 @@ import { inStock, price } from '../utils/productStock'
 import type { StoreProduct } from './product'
 import type { PromiseType } from '../../../typings'
 import type { Resolver } from '..'
+import { withTax } from '../utils/taxes'
 
 export type Root = PromiseType<ReturnType<typeof StoreProduct.offers>>
 
+const getHighPrice = (
+  offers: Root,
+  options: { includeTaxes: boolean } = { includeTaxes: false }
+) => {
+  const availableOffers = offers.filter(inStock)
+  const highOffer = availableOffers[availableOffers.length - 1]
+  const highPrice = highOffer ? price(highOffer) : 0
+  if (!options.includeTaxes) {
+    return highPrice
+  }
+
+  return withTax(highPrice, highOffer.Tax, highOffer.product.unitMultiplier)
+}
+
+const getLowPrice = (
+  offers: Root,
+  options: { includeTaxes: boolean } = { includeTaxes: false }
+) => {
+  const [lowOffer] = offers.filter(inStock)
+
+  const lowPrice = lowOffer ? price(lowOffer) : 0
+
+  if (!options.includeTaxes) {
+    return lowPrice
+  }
+
+  return withTax(lowPrice, lowOffer.Tax, lowOffer.product.unitMultiplier)
+}
+
 export const StoreAggregateOffer: Record<string, Resolver<Root>> & {
   offers: Resolver<Root, any, Root>
 } = {
-  highPrice: (offers) => {
-    const availableOffers = offers.filter(inStock)
-    const highOffer = availableOffers[availableOffers.length - 1]
-
-    return highOffer != null ? price(highOffer) : 0
-  },
-  lowPrice: (offers) => {
-    const [lowOffer] = offers.filter(inStock)
-
-    return lowOffer ? price(lowOffer) : 0
-  },
+  highPrice: (offers) => getHighPrice(offers),
+  lowPrice: (offers) => getLowPrice(offers),
+  lowPriceWithTaxes: (offers) => getLowPrice(offers, { includeTaxes: true }),
```

Comment:
This { includeTaxes: true } is used only inside the getLowPrice function. It's a way to reuse the function to calculate prices with and without taxes. The tax value comes from the Catalog, but whether the store will use taxes and in which section the taxes will be applied is defined in the hCMS. Because of this, both values are returned, and the front end chooses, based on the hCMS, which value should be used.